client/src/api.ts

```diff
@@ -55,6 +61,17 @@ export const pullGameState = async (): Promise<GameState> => {
   }
 };
 
+export const pullJoinGame = async (idGame: number): Promise<GameState> => {
```

Comment:
`pullJoinGame`?

Reply:
All our get calls are prefixed by pull, so I wanted to stay in this logic.
But i can rename to joinGame

Reply:
Disregard then